suite/tests/linux/clone.c

```diff
@@ -48,6 +48,7 @@
 #include <errno.h>
 
 #include "tools.h" /* for nolibc_* wrappers. */
```

Comment:
Re the PR description: The issue is likely not specific to Ubuntu. It probably happens where the glibc version is older (so the struct is not defined)?

Either way, instead of trying to fit that into the concise PR title, maybe include the affected platforms in the PR description.

Reply:
Done, modulo internal builds.